Doc/library/importlib.metadata.rst

```diff
@@ -111,12 +118,31 @@ This package provides the following functionality via its public API.
 Entry points
 ------------
 
-The ``entry_points()`` function returns a collection of entry points.
+.. class:: EntryPoint
+
+    Details of an installed entry point (as described below).
+
+.. class:: EntryPoints
+
+    Details of a collection of installed entry points (as described below).
+
+.. function:: entry_points(**select_params)
+
+   Returns a :class:`EntryPoints` instance describing entry points for the
+   current environment. Any given keyword parameters are passed to
+   ``EntryPoints.select`` for comparison to the attributes of the
+   individual entry point definitions (as described below).
+
+   Note: it is not currently possible to query for entry points based on
+   their ``dist`` attribute (as different ``Distribution`` instances do
+   not compare equal, even if they have the same attributes)
+
 Entry points are represented by ``EntryPoint`` instances;
 each ``EntryPoint`` has a ``.name``, ``.group``, and ``.value`` attributes and
 a ``.load()`` method to resolve the value. There are also ``.module``,
 ``.attr``, and ``.extras`` attributes for getting the components of the
```

Comment:
Would it make more sense to put this information under the ``EntryPoint`` directive?
```suggestion
.. class:: EntryPoint

   Entry points are represented by ``EntryPoint`` instances;
   each ``EntryPoint`` has a ``.name``, ``.group``, and ``.value`` attributes and
   a ``.load()`` method to resolve the value. There are also ``.module``,
   ``.attr``, and ``.extras`` attributes for getting the components of the
   ``.value`` attribute, and ``.dist`` for obtaining information regarding
   the distribution package that provides the entry point.
```

Reply:
Applied locally.